universal/ubi8/Dockerfile

```diff
@@ -167,6 +166,9 @@ RUN dnf -y module enable container-tools:rhel8 && \
     dnf -y install podman buildah skopeo fuse-overlayfs && \
     dnf -y clean all --enablerepo='*'
 
+# shasum
+RUN dnf -y install shasum
```

Comment:
This part of the Dockerfile is the very last and that's where we install all the cloud specific tools. Since that's just a small CLI utility I think the best place would be [at the beginning of the base image](https://github.com/devfile/developer-images/blob/main/base/ubi8/Dockerfile#L23).

Reply:
OK, done.

Reply:
Thank you. The PR is approved. Let me know if you are able to merge it or if you want me to do it.